iroh-net/src/magicsock.rs

```diff
@@ -942,6 +986,52 @@ impl Inner {
 
         futures::future::poll_fn(|cx| self.poll_send_udp(addr, &transmits, cx)).await
     }
+
+    fn send_call_me_maybe(&self, derp_region: u16, dst_key: PublicKey) {
+        let mut endpoints = self.endpoints.lock();
+        let is_fresh = endpoints.fresh_enough();
+        if !is_fresh {
+            info!(
+                "want call-me-maybe but endpoints stale; restunning ({:?})",
+                endpoints.last_endpoints_time
+            );
+
+            let actor_sender = self.actor_sender.clone();
+
+            tokio::task::spawn(async move {
+                actor_sender
+                    .send(ActorMessage::ReStun("refresh-for-peering"))
+                    .await
+                    .unwrap();
```

Comment:
.ok() if we don't care if it succeeds or not?

Can this also be a try_send without the task?

Reply:
This was like this before.

I thought if we could do try_send, but am not sure - if the actor channel were full, the restun wouldn't happen. But might still happen anyway after a timeout? @dignifiedquire maybe knows more?

Reply:
We shouldn't `unwrap`. `try_send` makes more sense to me, given this is non critical to do

Reply:
Removed the unwrap but kept the task for now.